### DISCUSSION TITLE
How to fix a problem with Windows 11

### DISCUSSION THREAD

**Discussion by Xeybl** [alvr-org/ALVR#1115] [Q&A] [answered]
On Windows 11, when trying to run alvr + oculus quest 2, the picture freezes and that's it. Consumption monitoring disappears on the laptop. If you try to launch on Windows 10, everything works perfectly. Everything works through the standard Oculus program.

**Comment by xDShot** [comment]
What card? What ALVR settings?

**Reply by Xeybl** [reply]
standart settings (cable) + 90 fps. On Windows 10, I played quietly at 120 fps. Card - 1050 TI (Mobile)

**Reply by xDShot** [reply]
I assume it's something to do with switchable graphics. If there's nvidia panel, set it to performance mode or force alvr to switch to discrete card.

**Reply by Xeybl** [reply]
Performance mode is enabled. When installing ALVR, a window about gtx 1050 ti (mobile) flew out, i.e. the application saw the video card.

**Reply by Xeybl** [reply]
?

**Comment by ColdIce1605** [comment, marked as answer]
yeah this is likely do to alvr or steamvr using different gpus or just one using integrated which is not going to work

**Reply by ColdIce1605** [reply]
could also be the windows 11 streamvr bug I've heard awhile back but thats been so long I'm not sure if that is what it was anymore.

**Reply by Xeybl** [reply]
Thanks, it works. The problem was in the first versions of Windows 11. At the moment, the fix for the problem has already been released.
